client/landing/gutenboarding/stores/domain-suggestions/index.ts

```diff
@@ -2,9 +2,5 @@
  * External dependencies
  */
 import { DomainSuggestions } from '@automattic/data-stores';
-import { getSuggestionsVendor } from 'calypso/lib/domains/suggestions';
```

Comment:
Should the function imported here be removed in favour of the one in the domain suggestion data-store? It feels like a bit of a duplicate

Reply:
> Should the function imported here be removed in favour of the one in the domain suggestion data-store? It feels like a bit of a duplicate

Yes, the choice of which vendor to use falls upon the responsibility of whoever is using the `<DomainPicker>` component.

The domain suggestions data store merely uses a default `variation2_front` vendor value.